File: c++/wikipedia/wiki_4.cpp
#include <iostream>

using namespace std;

int main(void)
{
	int n;
	int biggest=0;

	cout<<"masukan angka";
	cin>>n;

	while(n--){
		int temp;
		cout<<"masukan angka:";
		cin>>temp;
		if(temp > biggest)
			biggest=temp;
	}
	cout<<"biggest adalah:"<<biggest<<endl;
}

File: perl/wikipedia/wiki_4.pl
#!/usr/bin/perl

use strict;
use warnings;

my $max=0;
my $counter=0;
print "masukan angka limit:";
my $number=<STDIN>;
while($counter < $number){
	print "masukan angka ke-$counter:";
	my $temp=<STDIN>;
	if($temp > $max){
		$max=$temp;
	}
	$counter++;
}
print "angka terbesarnya adalah ",$max;

